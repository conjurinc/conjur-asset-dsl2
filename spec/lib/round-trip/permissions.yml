- !permissions
  resource: !variable foobar
  privilege: [ read, execute ]
  members:
  - !member
    role: !group developers
    admin: true
  - !group support
