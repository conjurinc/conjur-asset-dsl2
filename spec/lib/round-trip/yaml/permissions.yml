- !permit
  resource: !variable foobar
  privilege: [ read, execute ]
  role:
  - !member
    role: !group developers
    admin: true
  - !group support
