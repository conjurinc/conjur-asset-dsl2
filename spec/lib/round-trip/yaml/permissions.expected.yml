---
- !permit
  privilege:
  - read
  - execute
  resource: !variable
    id: foobar
  role:
  - !member
    role: !group
      id: developers
    admin: true
  - !member
    role: !group
      id: support
